Dockerfile

```diff
@@ -1,5 +1,10 @@
-FROM ruby:3.1-alpine
+FROM ruby:3.1
+
 COPY Gemfile Gemfile
-RUN bundle install --no-deployment
-COPY . .
+
+RUN gem install bundler && bundle install
+
+COPY lib lib
+COPY resources resources
```

Comment:
Is this nessessary in Dockerfile, or should that be part of `volumes` of docker-compose?

Reply:
Better to have it here so you can run without compose or any volume mounts if you want. It is mounted when using compose too though